### PR TITLE
Rust: Add Config for Separators

### PR DESCRIPTION
**Pull request by droogmic**

### CODE REVIEW

File: rust/osm2lanes/src/transform/lanes_to_tags.rs [hunk 1]
```diff
@@ -9,6 +9,7 @@ impl std::convert::From<DuplicateKeyError> for RoadError {
     }
 }
 
+#[non_exhaustive]
```
Comment (dabreegster) [inline rust/osm2lanes/src/transform/lanes_to_tags.rs:12]: I'm learning so much Rust by reviewing your PRs :) https://doc.rust-lang.org/reference/attributes/type_system.html#the-non_exhaustive-attribute

**Review by dabreegster — APPROVED** [review]
You're quite busy for being on holiday ;)

**Comment by droogmic** [comment]
> You're quite busy for being on holiday ;)

:D, had a quiet day, activities will start soon.

File: rust/osm2lanes/src/transform/lanes_to_tags.rs (hunk 1 above)
Comment (droogmic) [reply rust/osm2lanes/src/transform/lanes_to_tags.rs:12]: I'm also learning, so don't assume I know what I am doing :P

**Pull request merged**